client/mifare4.c

```diff
@@ -17,6 +17,52 @@
 #include "ui.h"
 #include "crypto/libpcrypto.h"
 
+AccessConditions_t MFAccessConditions[] = {
+	{0x00, "rdAB wrAB incAB dectrAB"},
```

Comment:
Suggest `dec` instead of `dectr`

Reply:
decrement and transfer. i think how to short write it

Reply:
leave? or change? what do you think?

Reply:
ah yes, I forgot. And Restore as well. Why use abbreviations at all? What about the wording from the NXP documents:
```
     {0x00, "key A|B read; key A|B write; key A|B increment; keyA|B increment, transfer, restore"},
```